anyway/parsers/location_extraction.py

```diff
@@ -138,6 +138,10 @@ def geocode_extract(location):
         gmaps = googlemaps.Client(key=secrets.get("GOOGLE_MAPS_KEY"))
         geocode_result = gmaps.geocode(location, region="il")
         if geocode_result is None or geocode_result == []:
```

Comment:
`if not geocode_result:` is both more idiomatic (PEP-8) and avoids subtle bugs (in case it happens to be a tuple, for instance).

Reply:
(Oops, not your code! but you are welcome to mend it, if you want to :)

Reply:
will do :)